index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Mohasin Haque">
+    <meta name="description"
+        content="FAST UI a CSS library with pre-defined classes for quick creation of your projects. Now build projects more faster by just in single click by integrating this FAST UI in your code.">
+    <link rel="icon" href="https://img.icons8.com/ios-glyphs/50/000000/fast-track.png">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css">
```

Comment:
you can include this in main.css. This way you don't have to include this in every page

Reply:
Done!